### PR TITLE
updated libraries and plugins for 5.8.0 #1216

### PR DESCRIPTION
**Pull request by btootomoyuk**
Please review https://github.com/terasolunaorg/terasoluna-gfw/issues/1216 .

### CODE REVIEW

File: terasoluna-gfw-parent/pom.xml [hunk 1]
```diff
@@ -489,7 +489,7 @@
     <org.mybatis.version>3.5.11</org.mybatis.version>
     <org.mybatis.spring.version>3.0.1</org.mybatis.spring.version>
     <!-- == MapStrcut== -->
-    <mapstruct.version>1.5.3.Final</mapstruct.version>
+    <mapstruct.version>1.5.2.Final</mapstruct.version>
```
Comment (btkatoufm) [inline terasoluna-gfw-parent/pom.xml:492]: - [ ] mapstruct should be 1.5.3

File: terasoluna-gfw-parent/pom.xml [hunk 2]
```diff
@@ -515,7 +515,7 @@
     <!-- == Other == -->
     <jakarta-el.version>5.0.1</jakarta-el.version>
     <jakarta-inject.version>2.0.1</jakarta-inject.version>
-    <jakarta-jsp.version>3.1.0</jakarta-jsp.version>
+    <jakarta-jsp.version>3.0.0</jakarta-jsp.version>
```
Comment (btkatoufm) [inline terasoluna-gfw-parent/pom.xml:518]: - [ ] jakarta.servlet.jsp-api should be 3.1.0

Comment (btootomoyuk) [reply terasoluna-gfw-parent/pom.xml]: Fixed in the following commit.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1217/commits/bc71e46813aee1b430b24849e1a24cdea8985423

File: terasoluna-gfw-parent/pom.xml (hunk 1 above)
Comment (btootomoyuk) [reply terasoluna-gfw-parent/pom.xml]: Fixed in the following commit.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1217/commits/bc71e46813aee1b430b24849e1a24cdea8985423

**Pull request merged**